spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +71,16 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	@PostConstruct
+	public void deduceModuleArguments() throws Exception {
```

Comment:
Actually, I think we don't want the concept of label to propagate down to the launcher. That was the rationale behind 2 corresponding arrays. The order is important, and relying on it, no label is necessary.